blog/content/post/2020-12-09-OSFC_2020.md

```diff
@@ -0,0 +1,198 @@
+
+* [UEFI compliance testing](https://vimeo.com/488138727) by Heinrich Schuchardt
+
+These guys are doing great stuff for ARM and RISC-V with tools that are worthy
+of mentioning, e.g., Platform Initialization SCT and Firmware Test Suite (FWTS).
+The overall quality of UEFI-compliance test results is impressive. We liked the
+idea of UEFI Self Certification Tests, UEFI SCT compliance across various UEFI
+implementations for different platforms (QEMU, x86, RISC-V). Kudos to [U-Boot](https://github.com/u-boot/u-boot) for your work.
+
+* [coreboot Thermal Management on Intel based systems](https://vimeo.com/488143663) by Sumeet Pawnikar, Timothy Wawrzynczak
+
+Intel works on this contribution to enable efficient thermal management for
+Google Chromebooks. It is an important topic for all types of devices,
+especially those focused on energy saving, e.g., mobile platforms. The
+presenters discussed the added ACPI code and explained the code tree with its
+structure. Kudos to [Intel](https://software.intel.com/content/www/us/en/develop/articles/coreboot.html) for your piece of work.
+
+* [Bootling Linux more safely with safeboot](https://vimeo.com/488144473) by [Trammell Hudson](@qrs)
+
+Trammel, the founder, and contributor of safeboot, presented a convenient talk
+around configuring UEFI Secure Boot tools. Kudos to Trammel Hudson for his work [safeboot](https://safeboot.dev/)
+
+* [PRM: SMM Goes on a Diet](https://vimeo.com/488136678) by Michael Kubacki
+
+Great, technical talk on System Management Mode.  A few noticeable facts:
+* some hardware resources can only be accessed in SMM
+* RAS relies on SMM and SMI handlers
+* UEFI Capsule Update and Authenticated Variables
+* SMM is related to OCP and server use cases https://www.youtube.com/watch?v=mu3DRLM1dPA
+Kudos to Microsoft and Intel for your [joint effort](https://www.phoronix.com/scan.php?page=news_item&px=Microsoft-PRM-OSFC-2020).
+
+* [Building an alternative ecosystem to UEFI](https://vimeo.com/488136082) by John Looney
+
+John presented Facebook build system, stages of industrialization (Prototype ->
+EVT -> DVT -> PVT -> Mass production), and testing (1000 reboots test are
+Facebook standard for DRAM init). According to Facebook: it's 2x faster to
+enable the platform on coreboot than UEFI. Good to hear that! Kudos to
+[Facebook](https://github.com/facebook) for insight into your work.
+
+* [System Transparency and it's bootloader implementation stboot](https://vimeo.com/488128723) by [Jens Drenhaus](https://twitter.com/jens_drenhaus)
+
+The talk was about stboot LinuxBoot distribution. Jens described swtpm testing
+and D-RTM (Intel version) use case for system provisioning with tboot as an
+alternative for TrenchBoot. Big kudos to [9elements](https://9elements.com/) for
+your project.
+
+* [kexec based bootloaders/fast rebooting: Boon or Bane](https://vimeo.com/488150148) by
+[Bhupesh Sharma](https://twitter.com/bhupesh_sharma)
```

Comment:
@KBanecka kexec is the theme of various boot processes, even OpenPOWER use it for Secure Boot. @miczyg1 can you help put something based on my notes?

Reply:
`* kexec based bootloaders are becoming a very popular and kexec itself are very capable (can boot Linux, Windows, Xen and more)`
`* kexec still is not robust, there is still a threat of DMA occurrence during kexec kernel loading`
`* not all Linux drivers are kexec-ready, they need a shutdown method to avoid undesired DMA and be relaunched correctly in new kernel`

Reply:
dodane